src/components/audit/filters/action.vue

```diff
@@ -105,8 +106,34 @@ export default {
 </script>
 
 <style lang="scss">
-.audit-filters-action-category {
-  // Not all browsers support styling an <option> element this way.
-  font-weight: bold;
+#audit-filters-action {
+  select {
+    appearance: none;
+    cursor: pointer;
+    padding-top: 23px;
+    padding-right: 25px;
+    height: 48px;
+  }
```

Comment:
Should some of these styles apply to `<select>` elements outside this component? I'm thinking of `ProjectSort`, for example. I wonder whether we should create a generic `<single-select>` component at some point.

Reply:
https://github.com/getodk/central-frontend/issues/1240